data/categories/vp_vehicles_parts.yml

```diff
@@ -1171,14 +1171,50 @@
   - vehicle_engine_part_features
 - id: vp-1-4-6-5
   name: Ignition Components
-  children: []
+  children:
+  - vp-1-4-6-5-1
+  - vp-1-4-6-5-2
+  - vp-1-4-6-5-3
+  - vp-1-4-6-5-4
   attributes:
   - color
   - item_condition
   - item_material
   - manufacturer_type
   - pattern
   - vehicle_engine_part_features
+- id: vp-1-4-6-5-1
+  name: Glow Plugs
+  children: []
+  attributes:
+  - item_condition
+  - item_material
+  - manufacturer_type
+  - vehicle_engine_part_features
+- id: vp-1-4-6-5-2
+  name: Ignition Coils
+  children: []
+  attributes:
+  - item_condition
+  - item_material
+  - manufacturer_type
+  - vehicle_engine_part_features
+- id: vp-1-4-6-5-3
+  name: Ignition Leads
+  children: []
+  attributes:
+  - item_condition
+  - item_material
+  - manufacturer_type
+  - vehicle_engine_part_features
+- id: vp-1-4-6-5-4
+  name: Distributor & Parts
+  children: []
+  attributes:
+  - item_condition
+  - item_material
+  - manufacturer_type
+  - vehicle_engine_part_features
```

Comment:
I would split these in two for better product classification, both at level 6:
- `Distributors`
- `Distributor Parts`